components/demo/demo-snippet.js

```diff
@@ -13,6 +13,7 @@ class DemoSnippet extends LitElement {
 			overflowHidden: { type: Boolean, reflect: true, attribute: 'overflow-hidden' },
 			_code: { type: String },
 			_dir: { type: String, attribute: false },
+			_isFrench: { type: Boolean, reflect: false },
```

Comment:
This was all written before Lit 2, but all of these `_` properties could use the new state stuff since they don't need to be "public" properties.

```suggestion
			_isFrench: { state: true },
```